dotcom-rendering/src/web/components/Liveness.importable.tsx

```diff
@@ -44,15 +44,15 @@ function insert(html: string, switches: Switches) {
 	// Shouldn't we sanitise this html?
 	// We're being sent this string by our own backend, not reader input, so we
 	// trust that the tags and attributes it contains are safe and intentional
-	const maincontent = document.querySelector<HTMLElement>('#maincontent');
-	const latestBlock = document.querySelector('#maincontent :first-child');
-	if (!latestBlock || !maincontent) return;
-	maincontent.insertBefore(fragment, latestBlock);
+	const blogBody = document.querySelector<HTMLElement>('#liveblog-body');
+	const latestBlock = document.querySelector('#liveblog-body :first-child');
```

Comment:
We’re already doing this below!

```suggestion
	const latestBlock = blogBody.querySelector(':first-child');
```

Reply:
This suggestion causes typescript errors so I've made the change locally